<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlackBox</class>
 <widget class="QMainWindow" name="BlackBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BlackBox</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>46</width>
          <height>13</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>46</width>
          <height>13</height>
         </size>
        </property>
        <property name="text">
         <string>HostApi:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="apiBox"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>41</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>41</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Input:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="devBox"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Gate:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="gateSlider">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>-64</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="gateLabel">
        <property name="text">
         <string>-64dB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>16</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>16</height>
         </size>
        </property>
        <property name="text">
         <string>Level:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>0</number>
        </property>
        <property name="value">
         <number>-100</number>
        </property>
        <property name="format">
         <string>%vdB</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
